core-bundle/src/Resources/contao/classes/DataContainer.php

```diff
@@ -1312,6 +1308,66 @@ public function invalidateCacheTags()
 		$cacheManager->invalidateTags($tags);
 	}
 
+	private function addPtableTags($strTable, $intId, &$tags)
+	{
+		if (empty($GLOBALS['TL_DCA'][$strTable]['config']['ptable']))
+		{
+			return;
```

Comment:
Here would be the general tag, right?

Reply:
Yes. Added in 5b4a03d8a5d5b6ea8b32c5375832499276e3eb1f.